.github/workflows/tenzir.yaml

```diff
@@ -616,7 +616,7 @@ jobs:
               # - The http-parser package is disabled on Hoembrew, which means
               #   we cannot build the web plugin on macOS anymore. Last checked
               #   on 2024-01-31.
-              - plugins/[^(parquet)(web)]*
+              - plugins/[^(parquet)(web)(sqs)]*
```

Comment:
This could really use a comment explaining _why_ we disable sqs on macOS, similar to the comments above for parquet and web.

Reply:
Does that work for you?